src/components/Book.js

```diff
@@ -1,14 +1,37 @@
 import PropTypes from 'prop-types';
+import '../styles/Book.css';
 
 function Book({ book, handleRemoveBook }) {
-  const { id, title, category } = book;
+  const { title, category } = book;
+
   return (
-    <tr>
-      <td>{id}</td>
-      <td>{title}</td>
-      <td>{category}</td>
-      <td><button type="button" onClick={handleRemoveBook}>Remove book</button></td>
-    </tr>
+    <>
+      <div className="item" id="book">
+        <div className="left">
```

Comment:
- [ ] this is a reusable component that is being used in an iteration, the id=book is not unique in this case. You can add uniqueness by concatenating it with the book id e.g` id={`book${book.id}`} `
- [ ] The container holding this components details should correspond to the design
- [ ] The font sizes should match the design